locallibs/userpkg.py

```diff
@@ -88,6 +89,9 @@
     /usr/bin/killall DirectoryService 2>/dev/null || /usr/bin/killall opendirectoryd 2>/dev/null
 """
 
+PI_CREATEHOMEDIR = """
+/usr/sbin/createhomedir -c
+"""
```

Comment:
Using createhomedir will only work when a package is installed to the current boot volume. All the other options for pycreateuserpkg work no matter if the package target is the current boot volume or an alternate volume (as when installing into an AutoDMG image, or via a Imagr or DeployStudio or Bootstrappr workflow). I'm hesitant to merge a feature that works only sometimes.

Reply:
In the latest 10.13.4 beta 4 there is also a new behavior to createmobileaccount that prompts in the cli run for an admin account to set the SecureToken. There is a `-D` option to suppress it. There should be some logic to detect the OS and act accordingly.

-Eric

Reply:
`createmobileaccount` and `createhomedir` are different tools; I don't know if there are similar new issues in 10.13.4 that would affect `createhomedir`.

Reply:
I will put in into the if block, under the killall statements